src/components/search/types.tsx

```diff
@@ -0,0 +1,4 @@
+export declare interface SearchProps {
```

Comment:
You do not need to use `declare` for your own types.
https://www.geeksforgeeks.org/explain-when-to-use-declare-keyword-in-typescript/
It will work without it, just`export interface SearchProps`

Reply:
One more interesting article about this https://javascript.plainenglish.io/purpose-of-declare-keyword-in-typescript-8431d9db2b10

Reply:
Fixed